src/Chunk.h

```diff
@@ -439,7 +439,12 @@ class cChunk :
 	void SetAlwaysTicked(bool a_AlwaysTicked);
 
 	// Makes a copy of the list
-	cClientHandleList GetAllClients(void) const {return m_LoadedByClient; }
+	cClientHandleList GetAllClients(void) const
+	{
+		cClientHandleList copy;
+		std::copy(m_LoadedByClient.begin(), m_LoadedByClient.end(), std::back_inserter(copy));
+		return copy;
```

Comment:
Ohhh.

I remember there being something about initialising containers based on ranges? So could you do `cClientHandleList copy(m_Loaded.begin(), m_Loaded.end())`?